doc/paradox/upgrade.md

```diff
@@ -30,6 +30,19 @@ Latest changes
 See @ref:[the generated tables of incompatible changes](compat-reports.md)
 
 
+Upgrade from 3.3 to 3.4
+-----------------------
+
+### ResultConverter
+``ResultConverter`` is a utility to convert between values handled in jdbc and values handled in
```

Comment:
Doesn't Slick already have a few such things? How does this differ?

Reply:
I have modified it. This is change log.

Reply:
>  If there is a justification to keep both modes (as separate methods, whether or not we deprecate the old mode), I think a good naming convention would be to suffix with Batch (e.g. insertAllBatch).

I think there is a reason to have both methods. The existing method has performance issues, but requires a smaller communication buffer. On the other hand, if there are inserting a very large number of values, the new method may be prone to communication errors because of its large sql statements.

Reply:
Yes but maybe make it clear why it was added given that there are other similar things

Reply:
> In order to support issuing insert statements with multiple values in Slick 3.4
(`++=` is a batch call of multiple single insert statement), we have made this interface available for offsets.
If you have extended a profile and used the `set` method of this interface,
you should specify 0 for this offset unless the number of variable placeholders in the SQL dynamically changes.

The reason is this, but is it not clear?